marketplacetests/marketplace_gaia_test.py

```diff
@@ -18,6 +23,40 @@ def setUp(self):
         # Use this to override the Marketplace app version
         self.MARKETPLACE_DEV_NAME = 'Marketplace'
 
+    def install_in_app_payments_test_app(self):
+
+        self.homescreen = Homescreen(self.marionette)
+
+        if not self.apps.is_app_installed(self.APP_NAME):
+
+            marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
+            marketplace.launch()
+            details_page = marketplace.navigate_to_app(self.APP_NAME)
+            details_page.tap_install_button()
+
+            # Confirm the installation and wait for the app icon to be present
+            confirm_install = ConfirmInstall(self.marionette)
+            confirm_install.tap_confirm()
```

Comment:
Damn, that's because I also made a change in <code>from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall</code>, but it didn't get through